lib/commands/file-movement.js

```diff
@@ -257,7 +297,10 @@ async function pullFromRealDevice (device, remotePath, isFile) {
         : await zip.toInMemoryZip(dstPath);
       return Buffer.from(buffer).toString('base64');
     } finally {
-      await exec('umount', [mntRoot]);
+      // Rarely umount fails, so retry once
+      retry(2, async function () {
```

Comment:
I got below error once even though the path was the same as succeeded case. So, let's retry once if umount fails.

```
[XCUITest] Umount has failed, so not removing '/var/folders/34/2222sh8n27d6rcp7jqlkw8km0000gn/T/201951-20388-1qk86jx.iyxy'
[debug] [W3C (d4c663dc)] Encountered internal error running command: Error: Command 'umount /var/folders/34/2222sh8n27d6rcp7jqlkw8km0000gn/T/201951-20388-1qk86jx.iyxy' exited with code 1
[debug] [W3C (d4c663dc)]     at ChildProcess.<anonymous> (/Users/kazu/GitHub/appium/node_modules/teen_process/lib/exec.js:94:19)
[debug] [W3C (d4c663dc)]     at ChildProcess.emit (events.js:200:13)
[debug] [W3C (d4c663dc)]     at maybeClose (internal/child_process.js:1021:16)
[debug] [W3C (d4c663dc)]     at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
[HTTP] <-- POST /wd/hub/session/d4c663dc-cf1e-4cfe-bf08-ad44edd4e621/appium/device/push_file 500 3779 ms - 699
[HTTP]
```